### DISCUSSION TITLE
v0.5-Alpha

### DISCUSSION THREAD

**Discussion by U-C-S** [U-C-S/Hurl#26] [Announcements]
- Add Acrylic Background, Native Rounded Corners and Shadows
- Deprecate `Hurl.Settings` (for now, directly edit the `UserSettings.json`)
- Requires `.NET 6 Desktop Runtime` (.netf48 version will be released for stable v0.5)

### Issues
- Untested in Win10 (might crash due to Rounded Corners thingy which is only supported in win11 or Acrylic Background)
- Hurl needs to be manually closed after Selecting a browser

## What's Changed [AUTO GENERATED]
* Project Restructure - The Seperation by @U-C-S in https://github.com/U-C-S/Hurl/pull/13
* Backport dev to dotnet 6 branch by @U-C-S in https://github.com/U-C-S/Hurl/pull/16
* Add WPFUI and Acrylic Background  by @U-C-S in https://github.com/U-C-S/Hurl/pull/22
* Updates for #22 by @U-C-S in https://github.com/U-C-S/Hurl/pull/23

**Full Changelog**: https://github.com/U-C-S/Hurl/compare/v0.3.0...v0.5.0-alpha-1

<hr /><em>This discussion was created from the release <a href='https://github.com/U-C-S/Hurl/releases/tag/v0.5.0-alpha-1'>v0.5-Alpha-1 - Acrylic Background</a>.</em>

**Comment by nitin2953** [comment]
Corners are overflowing on Windows 10

![image](https://user-images.githubusercontent.com/34852979/157581558-32a38255-1326-4b7d-a762-c29dd8667bd9.png)

**Reply by U-C-S** [reply]
Expected. since Native Rounded Corners are not supported in Win10. I have
to use some fallbacks (like resort to squary corners) ..... I'll see

**Reply by U-C-S** [reply]
Is the Acrylic Background working fine ??

**Reply by nitin2953** [reply]
Yes it is working fine. (after installing & launching first time it show some delay only in acrylic background)
Draggable area is too narrow, what do you think about making whole background draggable except buttons and bottom url

**Reply by U-C-S** [reply]
Ah... I forgot about that. I removed that code for experimenting.... Adding it back now

**Reply by U-C-S** [reply]
> Corners are overflowing on Windows 10

It seems we only few option to deal with this... Either
- Make corners squary to prevent that overflow look
- Remove Acrylic Background completely and fill it with normal colors instead
- Wait for WinAppSDK (forget this, This is only possible in future but App size will go up)

So, Naturally I'm going with first option

**Reply by nitin2953** [reply]
Border is rounded, can background be rounded in that way ?

**Reply by U-C-S** [reply]
That's the problem background acrylic can't rounded. It's possible only in win11.
